.travis.yml

```diff
@@ -11,7 +11,7 @@ r_github_packages:
   - jimhester/covr
 
 after_success:
-  - Rscript -e 'covr::codecov()'
+  - Rscript -e 'covr::codecov(line_exclusions = "compat-purrr.R")'
```

Comment:
Would you mind using exclusion markers in compat-purrr.R and keep this as is? Or is there another way to specify exclusions, via config file or the like?